data/human/human missions.txt

```diff
@@ -6893,3 +6893,51 @@ mission "Returning to Paradise"
 			action
 				payment 120000
 			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`
+
+mission "Hogshead's Dilemma"
+	invisible
+	source
+		near Sol 1 10
+	to offer
+		random < 5
+	on offer
+		conversation
+			`While passing through the spaceport, you notice a few energetic pilots gathered around a computer terminal, invested in some sort of conversation. One of the pilots speaks louder than the rest, sometimes getting looks from others in the spaceport. "You've fooled me before, and I reckoned you were this time! And here I thought them dang things were old news!" The pilot lets out a hearty chuckle. "Even nowadays them Firkins are still getting names!" After another minute or two, the group walks away from the terminal and out of your sight.`
+			choice
+				`	(Go see what this is about.)`
+				`	(Ignore it.)`
+					decline
+			`	You walk up to the computer terminal, and on it is displayed a news article with a title that reads "Popular Votes For Ship Name Poll - Unexpected Results." The article is about a small transport company hearkening back to older times with a special transport voyage centered around a Hogshead space liner. Hogsheads have all but lost the charm they once had, but it seems a good number of pilots are eager to join in on the fun.`
+			`	For the event, the transport company decided to have a public poll to name the ship, and while initial suggestions were casual names such as "Stark Lady" or "Timelight", some of the most popular ones are actually a collection of joke names - ones you would imagine a toddler would have come up with. Apparently, the names have gotten enough attention and enjoyment out of pilots that the company decided to take the top five most popular results and hold a vote on them.`
+			`	Would you like to vote on the name as well?`
```

Comment:
just an idea but I think it could be nice if a thumbnail of a Hoshead was displayed, so the player knows what ship this is talking about?
it takes a bunch of time to find one even if you want to, in CCOR space so I wouldnt expect most ppl to know what ship we're talking about otherwise